tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+              'Expected credentialStatus.statusListCredential to be a URL.'
+            );
+            const {slc} = await getSlc({issuedVc});
+            should.exist(slc,
+              'Expected statusListCredential to resolve to a ' +
+              'Verifiable Credential');
+            testSlCredential({slCredential: slc});
+          });
+          it('When the URL is dereferenced, the resulting verifiable ' +
+            'credential MUST have type property that includes the ' +
+            'BitstringStatusListCredential value.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=When%20the%20URL%20is%20dereferenced%2C%20the%20resulting%20verifiable%20credential%20MUST%20have%20type%20property%20that%20includes%20the%20BitstringStatusListCredential%20value.';
+            const {slc} = await getSlc({issuedVc});
+            slc.should.have.own.property(
+              'type').to.be.an('array',
+              'Expected type to be an array.'
+            );
+            slc.type.should.include(
+              'BitstringStatusListCredential',
+              'Expected type to be BitstringStatusListCredential.'
+            );
+          });
+          it('statusSize MAY be provided.',
+            // Note that this feature (statusSize) is currently at RISK
+            async function() {
+              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=statusSize%20MAY%20be%20provided.';
+              if(issuedVc.credentialStatus.statusSize) {
+                it('If present, ' +
+                  'statusSize MUST be an integer greater than zero.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20statusSize%20MUST%20be%20an%20integer%20greater%20than%20zero.';
+                  issuedVc.credentialStatus.statusSize.should.be.a('number',
+                    'Expected statusSize to be an integer.');
+                  // TODO test for integer
+                  assert(Number.isInteger(
+                    issuedVc.credentialStatus.statusSize),
+                  'Expected statusSize to be an integer.');
+                  issuedVc.credentialStatus.statusSize.should.be.gt(0,
+                    'Expected statusSize to be >= 0.');
+                });
+                if(issuedVc.credentialStatus.statusSize.gt(1)) {
+                  it('If statusSize is provided and is greater than 1, ' +
+                    'then the property credentialStatus.statusMessage ' +
+                    'MUST be present',
+                  async function() {
+                    this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20provided%20and%20is%20greater%20than%201%2C%20then%20the%20property%20credentialStatus.statusMessage%20MUST%20be%20present';
+                    issuedVc.credentialStatus.should.have.own.property(
```

Comment:
do we need to check statusSize here? the statement suggests that we do.

Reply:
The statement is already in a `if` that checks the size